opendevin/core/config.py

```diff
@@ -418,7 +418,7 @@ def get_llm_config_arg(llm_config_arg: str):
 
     ```
     [gpt-3.5-for-eval]
-    model = 'gpt-3.5-turbo'
+    model = 'gpt-4o'
```

Comment:
Is this right? It's under `[gpt-3.5-for-eval]`

Reply:
Ah, good catch, thank you!